src/Hocon.Configuration/Config.cs

```diff
@@ -54,27 +66,12 @@ public Config(HoconRoot root) : base(root?.Value, root?.Substitutions ?? Enumera
         /// <summary>
         ///     The configuration used as a secondary source.
         /// </summary>
-        public Config Fallback { get; private set; }
+        public Config Fallback { get; }
```

Comment:
Fallback should never change. It is either set, or not set for given config.

Reply:
Great idea.